modules/dhcp/dhcp_api.rb

```diff
@@ -67,8 +74,10 @@ class Proxy::DhcpApi < ::Sinatra::Base
   # returns a record for a mac address
   get "/:network/mac/:mac_address" do
     param_network = params[:network]
-    param_mac = params[:mac_address].downcase unless params[:mac_address].nil?
+    param_mac = params[:mac_address].downcase if params[:mac_address]
     begin
+      server.validate_supported_address(params[:network])
+      validate_mac(params[:mac_address]) if params[:mac_address]
```

Comment:
Small problem, you downcase parameter into `param_mac` variable and then validate param. Not a big deal but if you could validate `param_mac` instead, that is cleaner.